src/hooks/useFetchUniqueTokens.ts

```diff
@@ -141,35 +153,13 @@ export default function useFetchUniqueTokens({
       setShouldFetchMore(false);
       (async () => {
         // Fetch more Ethereum tokens until all have fetched
-        let tokens = (
-          await fetchMore({
-            network,
-            // If there are stored tokens in storage, then we want
-            // to do a background refresh.
-            page: hasStoredTokens ? 0 : 1,
-            uniqueTokens: hasStoredTokens ? [] : uniqueTokens,
-          })
-        ).filter((token: any) => token.familyName !== 'POAP');
-
-        // Fetch poaps
-        const poaps = await fetchPoaps(address);
-        if (poaps) {
-          tokens = [...tokens, ...poaps];
-        }
-
-        // Fetch Polygon tokens until all have fetched
-        const polygonTokens = await fetchMore({ network: Network.polygon });
-        tokens = [...tokens, ...polygonTokens];
-
-        // Fetch recently registered ENS tokens (OpenSea doesn't recognize these for a while).
-        // We will fetch tokens registered in the past 48 hours to be safe.
-        const ensTokens = await fetchEnsTokens({
```

Comment:
new ens names appear in simplehash api response ~immediately, tested this by registering 2 new names (only took 1 refresh for each to appear).